<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NameDialogCTC</class>
 <widget class="QDialog" name="NameDialogCTC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QToolButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>136</x>
     <y>160</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QToolButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>160</y>
     <width>86</width>
     <height>28</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>113</x>
     <y>70</y>
     <width>320</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="placeholderText">
    <string>请输入账户名称</string>
   </property>
  </widget>
  <widget class="QToolButton" name="closeBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>7</y>
     <width>24</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>账户名称：</string>
   </property>
  </widget>
  <widget class="QLabel" name="addressNameTipLabel1">
   <property name="geometry">
    <rect>
     <x>118</x>
     <y>112</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="addressNameTipLabel2">
   <property name="geometry">
    <rect>
     <x>148</x>
     <y>110</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="title_Label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>新建账户</string>
   </property>
  </widget>
  <zorder>title_Label</zorder>
  <zorder>okBtn</zorder>
  <zorder>cancelBtn</zorder>
  <zorder>nameLineEdit</zorder>
  <zorder>closeBtn</zorder>
  <zorder>label</zorder>
  <zorder>addressNameTipLabel1</zorder>
  <zorder>addressNameTipLabel2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
